modules/targetanalysis/src/target_loader.cpp

```diff
@@ -35,6 +66,7 @@ using namespace std;
 * }
 */
 
+/*
 class targetLoader{
```

Comment:
@chrishajduk84 
Commented out for now. Has various issues, most prominently being that it won't compile.
Please use pull requests from now on to submit code so that it can be reviewed (Admittedly this isn't something that we've really started doing yet and we'll need to set up some sort of system for dealing with pull requests).
If you create a pull request for this I'll comment on some of the other issues.